server/lib/buff_server_grpc/buff.pb.ex

```diff
@@ -0,0 +1,37 @@
+defmodule BuffServerGrpc.LoginRequest do
```

Comment:
this is all the generated stuff? the convention is to mark the extension as pb.ex so we'll know its generated?